go/runtime/host/protocol/connection.go

```diff
@@ -530,7 +530,7 @@ func (c *connection) InitHost(ctx context.Context, conn net.Conn) (*version.Vers
 	}
 
 	info := rsp.RuntimeInfoResponse
-	if ver := version.FromU64(info.ProtocolVersion); ver.MajorMinor() != version.RuntimeHostProtocol.MajorMinor() {
+	if ver := version.FromU64(info.ProtocolVersion); ver.Major != version.RuntimeHostProtocol.Major {
```

Comment:
can also remove `MajorMinor()` as here, and bellow in p2p are the only places where it's used.

Reply:
Actually we could probably just change it to use `.Major` but this would be incompatible with `1.0.0`.

Reply:
> Actually we could probably just change it to use `.Major` but this would be incompatible with `1.0.0`.

I understand. Are there any objections to bumping runtime host protocol version to 2.0.0?

Reply:
That statement was referring to the consensus protocol where this would be a breaking change. I don't think we need to bump the RHP version as this doesn't really break anything for the current protocol.

Reply:
> I don't think we need to bump the RHP version as this doesn't really break anything for the current protocol.

Yes, agreed.